test/phoenix/html/link_test.exs

```diff
@@ -24,4 +24,39 @@ defmodule Phoenix.HTML.LinkTest do
                    ~s[<a href="#" onclick="this.parentNode.submit(); return false;">hello</a>] <>
                    ~s[</form>]}
   end
+
+  test "button with post (default)" do
+
```

Comment:
No new lines after `test` and `before` (please check the other tests in the file). :)